### ISSUE TITLE
Mandatory CSS parameter order when drawing rectangles

### ISSUE THREAD

**Issue by fredrb** [raphamorim/origami.js#20]
When drawing a rectagle, it works if border parameters are like this:
```
origami('#canvas')
  .rect(0, 0, context.canvas.clientWidth, context.canvas.clientHeight, {
    background: 'lightblue',
    border: '4px solid #999'
  }).draw();
```

But if I swap them, such as:
```
origami('#canvas')
  .rect(0, 0, context.canvas.clientWidth, context.canvas.clientHeight, {
    background: 'lightblue',
    border: 'solid 4px #999'
  }).draw();
```

The exception `Uncaught TypeError: Failed to execute 'setLineDash' on 'CanvasRenderingContext2D': The 1st argument is neither an array, nor does it have indexed properties` is thrown.

If you check branch `bug/css_parameter_order` in my fork there are unit tests describing this behaviour.

**Comment by raphamorim** [comment]
That's true, I'm working on it by now.

**Comment by raphamorim** [comment]
Solved in: 5118a8eed777db59dd7453035f00302d269212e4

**Comment by fredrb** [comment]
:+1:
